server/controllers/Sessions.js

```diff
@@ -0,0 +1,66 @@
+import bcrypt from 'bcryptjs';
+import {
+  findUser,
+  generateToken,
+  serverResponse,
+  serverError,
+  expiryDate,
+  getUserAgent
+} from '../helpers';
+import models from '../database/models';
+
+const { Session } = models;
+
+/**
+ *
+ *
+ * @class Session
+ */
+class Sessions {
+  /**
+   *
+   *
+   * @static
+   * @param {object} req - request object
+   * @param {object} res - response object
+   * @memberof User
+   * @returns {json}  object
+   */
+  static async create(req, res) {
+    try {
+      const { userLogin, password } = req.body;
+      if (Object.keys(req.body).length < 1 || !userLogin || !password) {
+        return serverResponse(res, 400, {
+          message: 'invalid authentication details'
+        });
+      }
+      const user = await findUser(userLogin);
+      let verifyPassword;
```

Comment:
@henryade Could you please check if the request body is empty or either of the user login details is not present and return the appropriate response to the client?